cmd/namespace/use.go

```diff
@@ -127,12 +127,13 @@ func getNamespacesSelection(ctx context.Context) ([]utils.SelectorItem, error) {
 
 	namespaces := []utils.SelectorItem{}
 	for _, space := range spaces {
-
-		namespaces = append(namespaces, utils.SelectorItem{
-			Name:   space.ID,
-			Label:  space.ID,
-			Enable: true,
-		})
+		if space.Status == "Active" {
```

Comment:
Should we display also `Sleeping` spaces? In those spaces you can deploy things

Reply:
One option could be:
```
if space.Status == "Terminating" {
  continue
}

Reply:
```
% okteto list namespace
Namespace       Status
foo-jpf-okteto  Deleting
jpf-okteto      Sleeping
```

Isn't it `Deleting`?

Reply:
I think the state is `Deleting`: https://github.com/okteto/app/blob/master/backend/pkg/model/space.go#L31